src/utils/useHotspot.ts

```diff
@@ -129,6 +129,7 @@ const useHotspot = () => {
 
   const connectAndConfigHotspot = async (
     hotspotDevice: Device,
+    isDiagnostics?: boolean,
```

Comment:
could we use an options object here? That way when viewing a function call the intent is more clear
`connectAndConfigHotspot(hotspot, { isDiagnostics: true} )`

Reply:
Yeah, ill update it